Source/WebInspectorUI/UserInterface/Views/FindBanner.js

```diff
@@ -227,14 +229,17 @@ WI.FindBanner = class FindBanner extends WI.NavigationItem
 
     // Private
 
-    _inputFieldKeyDown(event)
-    {
-        if (event.keyIdentifier === "Shift")
-            this._searchBackwards = true;
-        else if (event.keyIdentifier === "Enter")
-            this._searchKeyPressed = true;
-    }
+    _inputFieldKeyDown(event) {
```

Comment:
style: the `{` should go on its own line (like it was before)

Reply:
Fixed in pr.